docs/workflows/manage-content.rst

```diff
@@ -9,6 +9,10 @@ There are multiple ways that users can manage Container content in repositories:
    2. Recursively :ref:`add<recursive-add>` or :ref:`remove<recursive-remove>` Container content.
    3. Copy :ref:`tags<tag-copy>` or :ref:`manifests <manifest-copy>` from source repository.
 
+.. note::
+   In a push repository it is not possible to perform any tagging, additon or removal of content via Pulp API.
+   However, it is possible to use push repository as a source repository in copy operations.
```

Comment:
```suggestion
   However, it is possible to use the push repository as a source repository in copy operations.
```

Reply:
Here we could actually use the indefinite article ("a").

Reply:
Yes, I had to reread the sentence again.